builder/llpcBuilderImplInOut.cpp

```diff
@@ -730,6 +763,15 @@ Value* BuilderImplInOut::ReadBuiltIn(
         LLPC_ASSERT(pIndex == nullptr);
         args.push_back((pVertexIndex != nullptr) ? pVertexIndex : getInt32(InvalidValue));
         break;
+    case ShaderStageFragment:
+        if (builtIn == BuiltInSamplePosOffset)
+        {
+            // Special case for BuiltInSamplePosOffset: pVertexIndex is the sample number.
+            args.push_back(pVertexIndex);
```

Comment:
Change to this:
// Special case ....
auto pSampleNum = pVertexIndex;
pVertexIndex = nullptr;
args.push_back(pSampleNum);

Otherwise, we mgith have to change pVertexIndex to pVertexIndexOrSampleNum. But I think it is somewhat verbose.